schemas/code_signature.yml

```
@@ -123,3 +123,15 @@
       description: >
         Date and time when the code signature was generated and signed.
       example: "2021-01-01T12:10:30Z"
+
+    - name: thumbprint
+      level: extended
+      type: keyword
+      short: Unique ID (sha256 hash) of the certificate .
```

Consistency with other fields?
```suggestion
      short: SHA256 hash of the certificate.
```

https://github.com/elastic/ecs/blob/77bb0b09ab84ec63c7d831269536a83551fb4d69/schemas/hash.yml#L63-L66